### DISCUSSION TITLE
Question about Code

### DISCUSSION THREAD

**Discussion by fmc5-git** [GarnesDoug/FVT-Baylor-University#4] [Q&A] [answered]
There are a lot of variations of 

AND PeriodBudget.AidYear = SUBSTR(PeriodBudget.NSCAidYear, 3, 2) || SUBSTR(PeriodBudget.NSCAidYear, 7, 2)

Is this correct? Isn't PeriodBudget.NSCAidYear a four character value?

**Comment by GarnesDoug** [comment, marked as answer]
Nope!  It is 20222023 format, I hope that helps.

![image](https://github.com/user-attachments/assets/e01bba17-be66-47b4-9942-11637b9ac05c)

**Reply by fmc5-git** [reply]
Thanks...I just got a copy of the actual completers list...I was ASSuming and that is what happens.

Sorry!!

**Reply by fmc5-git** [reply]
Also, thank you so much for providing this script! it's been extremely helpful. 

I was having issues with the script mainly because we switched over to Banner SaaS (Postgres) so I had to make some changes.

Matt

**Reply by fortrans** [reply]
Hi Matt, Did you see my question about creating the nsc.FVT table?  Did you run into the same issue?
